tests/onnx/test_onnx_v2.py

```diff
@@ -180,6 +180,7 @@ def test_values_override(self):
     ("ibert", "kssteven/ibert-roberta-base"),
     ("camembert", "camembert-base"),
     ("convbert", "YituTech/conv-bert-base"),
+    ("codegen", "Salesforce/codegen-350M-multi"),
```

Comment:
small detail but this line should be in between `camembert` and `convbert` similar to how it is in the `src/transformers/onnx/features.py` file

Reply:
Thanks for noticing @NouamaneTazi, to make the test more organised, it would be better to place it in with_past model list.